extensions/amp-byside-content/0.1/amp-byside-content.js

```diff
@@ -338,6 +389,22 @@ export class AmpBysideContent extends AMP.BaseElement {
   }
 }
 
+/**
+ * If scrolling is allowed for the iframe, wraps the element into a container
+ * that is scrollable because iOS auto expands iframes to their size.
+ * @param {!Element} element
+ * @return {!Element} The container or the iframe.
+ */
+function makeIOsScrollable(element) {
```

Comment:
there shouldn't be a need for this or `container_` anymore since iframe is now never scrollable. Also `amp-active` stuff should not be needed.

Reply:
You're right. Updated code to remove the `container_` and no longer use the `amp-active` class.